opsdroid/events.py

```diff
@@ -168,6 +199,10 @@ def __init__(self, text, *args, **kwargs):
         self.text = text
         self.raw_match = None
 
+    def __repr__(self):
+        """Override Message's representation so you can see the text when you print it."""
+        return f"<opsdroid.events.Message(text={self.text})>"
```

Comment:
I think this is the first f-string that has made it in since we dropped 3.5 🎉.

Reply:
All hail f-strings!